src/codewhisperer/models/constants.ts

```diff
@@ -123,7 +123,9 @@ export const welcomeCodeWhispererReadmeFileSource = 'resources/markdown/WelcomeT
 
 export const welcomeCodeWhispererCloud9Readme = 'resources/markdown/WelcomeToCodeWhispererCloud9.md'
 
-export const welcomeMessageKey = 'CODEWHISPERER_WELCOME_MESSAGE'
+export const welcomeMessageKeyPrevious = 'CODEWHISPERER_WELCOME_MESSAGE'
```

Comment:
Can you add a comment, explain that `welcomeMessageKeyPrevious` is a key for the welcome page of previous v1.91.0, `welComeMessageKeyShown` is a new key for the new getting started experience, it is used to track if user has seen the getting started page or not.

Reply:
Updated accordingly.

Reply:
`oldWelcomeMessageKey`